docs/docsite/rst/user_guide/playbooks_delegation.rst

```diff
@@ -19,21 +19,55 @@ You'll also want to read up on :doc:`playbooks_reuse_roles`, as the 'pre_task' a
 
 Be aware that certain tasks are impossible to delegate, i.e. `include`, `add_host`, `debug`, etc as they always execute on the controller.
 
+
 .. _rolling_update_batch_size:
 
 Rolling Update Batch Size
 `````````````````````````
 
-
 By default, Ansible will try to manage all of the machines referenced in a play in parallel.  For a rolling update use case, you can define how many hosts Ansible should manage at a single time by using the ``serial`` keyword::
 
 
     - name: test play
       hosts: webservers
-      serial: 3
+      serial: 2
+      gather_facts: False
+      tasks:
+      - name: task one
+        comand: hostname
+      - name: task two
+        command: hostname
+
+In the above example, if we had 4 hosts in the group 'webservers', 2
+would complete the play completely before moving on to the next 2 hosts::
+
+
+    PLAY [webservers] ****************************************
+
+    TASK [task one] ******************************************
+    changed: [web2]
```

Comment:
Are you showing something subtly here by having web2 before web1 for task 1, but not for task 2?  ...just curious...

Reply:
order is not set, it depends on how long the task takes on each host

Reply:
ok thanks for the explanation :-)